templates/hint.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { classes } from 'core/js/reactHelpers';
+
+export default function Hint (props) {
+  const {
+    _globals,
+    _hint
+  } = props;
+
+  return (
+    <div className={classes(["hint__inner"])}>
+
+    <button className={classes([
+              'hint__btn',
+              'js-hint-btn-popup',
+              (_hint._button._iconClass) && 'btn-icon',
+              (_hint._button.alignIconRight) && 'align-icon-right',
+              (_hint._button.text) && 'btn-text'
+            ])}
+            aria-label={classes([
+              (_hint._button.ariaLabel) ? _hint._button.ariaLabel : _globals._extensions._hint.openButtonText
+            ])}
```

Comment:
classes function is not needed here.
```suggestion
            aria-label={_hint._button.ariaLabel ? _hint._button.ariaLabel : _globals._extensions._hint.openButtonText}
```
You also probably want to make a variable at the top of the template called "buttonAriaLabel":
```js
  const buttonAriaLabel = _hint._button.ariaLabel 
    ? _hint._button.ariaLabel 
    : _globals._extensions._hint.openButtonText;
```
Then
```suggestion
            aria-label={buttonAriaLabel}
```